README.md

```diff
@@ -216,6 +232,123 @@ foreach ($rows as $row) {
 
 *Note that each row returned by the iterator is an associative array and matches the projection specified in your query.*
 
+### Observing Events
+
+To observe all events of a subject, call the `observeEvents` function with the subject as the first argument and an options object as the second argument. Set the `recursive` option to `false`. This ensures that only events of the given subject are returned, not events of nested subjects.
+
+The function returns an asynchronous iterator, which you can use e.g. inside a `foreach` loop:
+
+```php
+use Thenativeweb\Eventsourcingdb\ObserveEventsOptions;
+
+$events = $client->observeEvents(
+  '/books/42',
+  new ObserveEventsOptions(
+    recursive: false,
+  ),
+);
+
+foreach ($events as $event) {
+  // ...
+}
+```
+
+#### Observing From Subjects Recursively
+
+If you want to observe not only all the events of a subject, but also the events of all nested subjects, set the `recursive` option to `true`:
+
+```php
+use Thenativeweb\Eventsourcingdb\ObserveEventsOptions;
+
+$events = $client->observeEvents(
+  '/books/42',
+  new ObserveEventsOptions(
+    recursive: true,
+  ),
+);
+
+foreach ($events as $event) {
+  // ...
+}
+```
+
+This also allows you to observe *all* events ever written. To do so, provide `/` as the subject and set `recursive` to `true`, since all subjects are nested under the root subject.
+
+#### Specifying Bounds
+
+Sometimes you do not want to observe all events, but only a range of events. For that, you can specify the `lowerBound` option.
+
+Specify the ID and whether to include or exclude it:
+
+```php
+use Thenativeweb\Eventsourcingdb\Bound;
+use Thenativeweb\Eventsourcingdb\BoundType;
+use Thenativeweb\Eventsourcingdb\ObserveEventsOptions;
+
+$events = $client->observeEvents(
+  '/books/42',
+  new ObserveEventsOptions(
+    recursive: false,
+    lowerBound: new Bound('100', BoundType::INCLUSIVE),
+  ),
+);
+
+foreach ($events as $event) {
+  // ...
+}
+```
+
+#### Starting From the Latest Event of a Given Type
+
+To observe starting from the latest event of a given type, provide the `fromLatestEvent` option and specify the subject, the type, and how to proceed if no such event exists.
+
+Possible options are `wait-for-event`, which waits for an event of the given type to happen, or `read-everything`, which effectively behaves as if `fromLatestEvent` was not specified:
```

Comment:
Die Optionen heißen streng genommen nicht `wait-for-event` und `read-everything`, sondern `ObserveIfEventIsMissing::READ_EVERYTHING` und so 😊